src/map/clif.cpp

```diff
@@ -25550,6 +25550,108 @@ void clif_parse_macro_checker( int32 fd, map_session_data* sd ){
 #endif
 }
 
+/*==========================================
+ * macro user report client packet processing function
+ *------------------------------------------*/
+void clif_parse_macro_user_report(int32 fd, map_session_data *sd)
+{
+#if (PACKETVER_MAIN_NUM > 20230915)
```

Comment:
```suggestion
#if PACKETVER_MAIN_NUM >= 20230915
```
According to your struct definition.